policyengine_uk/variables/gov/hmrc/tax_free_childcare/conditions/tax_free_childcare_income_condition.py

```diff
@@ -0,0 +1,49 @@
+from policyengine_uk.model_api import *
+
+
+class tax_free_childcare_meets_income_requirements(Variable):
+    value_type = bool
+    entity = Person
```

Comment:
Could you add more comments tying to the law here? I couldn't tell if income is based on combined income with thresholds summed based on age, or requiring each spouse to meet their respective income conditions.